### DISCUSSION TITLE
[macOS] Legacy FlashPromptDialog.nib and PlayerUILocalizable.strings files?

### DISCUSSION THREAD

**Discussion by itlancer** [airsdk/Adobe-Runtime-Support#2341] [Q&A] [answered]
macOS AIR apps seems have legacy FlashPlayer files:
`MyApp.app/Contents/Frameworks/Adobe AIR.framework/Versions/1.0/Resources/en.lproj/FlashPromptDialog.nib`
`MyApp.app/Contents/Frameworks/Adobe AIR.framework/Versions/1.0/Resources/en.lproj/PlayerUILocalizable.strings`
for all languages.

AIR SDK have such files in directories:
`AIR_SDK/runtimes/air-captive/mac/Adobe AIR.framework/Versions/1.0/Resources/en.lproj/`

Does these files legacy or still needed for AIR applications?

**Comment by ajwfrost** [comment]
Hi

So looking at this, the .nib file was solely used for prompt dialogs that were requested by JavaScript running within the WebKit engine .. i.e. not actually relevant for recent versions of AIR.

The strings are being used still though - most of them anyway.

We'll drop the nib file from the build process and SDK....

thanks

**Reply by itlancer** [reply, marked as answer]
`FlashPromptDialog.nib` removed. Tested with latest AIR 51.1.2.2.
Thanks!
